content-src/components/DiscoveryStreamComponents/Hero/Hero.jsx

```diff
@@ -53,6 +54,16 @@ export class Hero extends React.PureComponent {
         source={truncateText(rec.domain, 22)} />
     ));
 
+    let list = (
+      <List
```

Comment:
Can you just confirm this won't be weird in terms of the telemetry? (i.e. will it think we have another list component or something?)